src/Tribe/Editor/Configuration.php

```diff
@@ -15,6 +16,45 @@ public function hook() {
 		add_filter( 'tribe_editor_config', array( $this, 'editor_config' ) );
 	}
 
+	/**
+	 * Setup the initial Event Object.
+	 *
+	 * @since TBD
+	 *
+	 * @todo @backend Improve this method and verify if this is the correct place for this to live.
+	 *
+	 * @param null|int|WP_Post $event
+	 *
+	 * @return array Setup the Data for the Event Post Object.
+	 */
+	public function get_event_object( $event = null ) {
+		$data = [
+			'is_new_post' => true,
+		];
+
+		if ( ! $event ) {
+			$event = tribe_get_request_var( 'post', false );
+		}
+
+		if ( $event ) {
+			$event = get_post( $event );
+		}
+
+		if ( ! $event instanceof  WP_Post) {
+			return $data;
+		}
+
+		$meta = get_post_meta( $event->ID );
+		$meta = array_map( static function( $value ) {
+			return count( $value ) === 1 ? reset( $value ) : $value;
+		}, $meta );
+
+		$data['is_new_post'] = false;
+		$data['meta'] = $meta;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: ca0e810cdd0274eb6520bcccd3614f00</sub>